### ISSUE TITLE
Tree testing for VPN

### ISSUE THREAD

**Issue by nicolebjelica** [mozilla/dpx-planning-and-intake#104]
### Description

Tree testing navigation for VPN 1.6+

### Status
Completed

**Comment by jess-cook03** [comment]
@isobelroe Is this something we need to keep or can we close this?

**Comment by isobelroe** [comment]
Yep can close this. The outcome of this was a report that was generated from a test run on UserZoom in Q4 2020: [Results from Tree Test on UserZoom](https://drive.google.com/file/d/1kEGBr8vYqgIQr1BCDjgFDmJ7wh-KlY-r/view?usp=sharing)
